HelpSource/Reference/Assignment.schelp

```diff
@@ -14,6 +14,50 @@ x = [1, 2, 3, 4].rotate(1);
 c = a + b;
 ::
 
+section:: Assignment in Declaration
+
+When declaring a variable, a value can be assigned to it immediately:
+
+code::
+(
+var x = 1;
+var y = -1;
+x + y
```

Comment:
This code is likely to mislead users in the same way we saw with #3606, if they run it one line at a time. Perhaps here you should use a longer variable name, or just use the global `x` and `y` directly

Reply:
ok, fixed. By default, the parentheses should enforce a region, but currently, the help viewer doesn't do this.

Reply:
Even if it is a region, I get the impression that lots of beginning users will run code one line at a time to see what it does at each step. That was my experience and IMO is encouraged by most of the examples use single letter variables and environment variables.

Reply:
makes sense! 
In this case, I have now changed it to declared variables, because that is the content of the documentation at this point.